packit_service/worker/testing_farm.py

```diff
@@ -110,6 +110,21 @@ def _payload(self, build_id: int, chroot: str) -> dict:
         if self.fmf_ref:
             fmf["ref"] = self.fmf_ref
 
+        artifacts = [
+            {
+                "id": f"{build_id}:{chroot}",
+                "type": "fedora-copr-build",
+            }
+        ]
```

Comment:
maybe I just don't get it correctly, but if the `built_packages` are present, we want to send only info about them without this, or am I wrong?

Reply:
I think that we need to send also info about the copr build, otherwise the tft wouldn't know where to look for specified packages.